### DISCUSSION TITLE
Auto-switch to and from tablet mode for foldables

### DISCUSSION THREAD

**Discussion by kaibagley** [thunder-app/thunder#1558] [Q&A] [answered]
Moved to feature request: https://github.com/thunder-app/thunder/issues/1560

Hi everyone,

Just wondering if there's a way to automatically switch on and off tablet mode for a foldable device (in my case Pixel 9 Pro Fold)

When on, the UI shows 2 squished columns in the front screen of the phone, and when off it looks much too wide on the main screen.

Thanks for the great app!

**Comment by micahmo** [comment, marked as answer]
Hey @kaibagley, thanks for the discussion question! I'm not aware of any such way to automatically toggle the tablet mode right now. But I think a foldable is a perfect use case for such a thing. Would you be willing to open a feature request for this idea? Thanks!

**Reply by kaibagley** [reply]
Done: https://github.com/thunder-app/thunder/issues/1560. Thanks for the reply!
